usecases/aggregate_node_data.py

```diff
@@ -0,0 +1,21 @@
+from typing import Union
+
+from deepdiff import DeepDiff
+from gateways.node_gateway import NodeGateway
+
+
+class AggregateNodeData:
+
+    def __init__(self, node_gateway: Union[NodeGateway, None] = None) -> None:
+        self.node_gateway = node_gateway or NodeGateway()
+
+    def aggregate(self) -> Union[bool, None]:
+        new_network = self.node_gateway.aggregate_network()
+        old_network = self.node_gateway.get_network()
+
+        networks_diff = DeepDiff(new_network, old_network)
+
+        if networks_diff:
+            return self.node_gateway.save_network(new_network)
```

Comment:
I'd say we would enter here "all the time". Should we store a hit counter to see if this diff is really necessary?

Reply:
I don't think so, as we introducing `up_since` instead of `uptime` for comparison

Reply:
I got it. But only if we collect data twice or more in the same second, right? Otherwise, even the `up_since` will be different. Maybe we should create a comparison method that receives a tolerance parameter in seconds (e.g. `compareNodes(node1, node2, tol=60)`). What do you think?

Reply:
`up_since` is supposed to be fixed for a running node. It's the timestamp when it started to run.
As we calculate it in runtime using `uptime`,  two instances of `AggregatedNode` for the same node can be different for milliseconds. That's why `up_since` is also `int` instead of `float`.
We still have the edge case where this milliseconds difference also change the integer part of the second but i don't think it's a problem.